src/main.cpp

```diff
@@ -99,6 +100,16 @@ void LogCallback(std::string const& msg) {
  *  /return Returns MStatus::kSucccess if everything went all right. */
 MStatus initializePlugin(MObject object)
 {
+	std::time_t current_time = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+	auto local_time = std::localtime(&current_time);
```

Comment:
This code can also go into the #ifndef, right? Since it's not used anywhere else..